pkg/apis/core/types.go

```diff
@@ -2279,6 +2279,24 @@ type Container struct {
 	// +featureGate=InPlacePodVerticalScaling
 	// +optional
 	ResizePolicy []ContainerResizePolicy
+	// Restart policy for the container to manage the restart behavior of each
+	// container within a pod.
+	// By default, if no value is assigned to this field, the restart behavior of
```

Comment:
This is somewhat confusingly worded (and hard to explain well).

How about:

"""
RestartPolicy defines the restart behavior of individual containers in a pod.  This field may only be set for init containers, and the only allowed value is "Always".  For non-init containers or when this field is not specified, the restart behavior is defined by the Pod's restart policy and the container type.

Reply:
applied